tools/common.py

```diff
@@ -7,7 +7,7 @@ def func_dump_pipeline(pipeline, noKey=False):
             output += key + "=" + str(value) + ";"
     return output.strip()
 
-def func_export_pipeline(pipeline_lst):
+def export_pipeline(pipeline_lst):
```

Comment:
export_pipeline() is still a vague name. I was scratching my head as to where we're exporting the pipeline. What would make this clear is something like export_pipeline_env_vars() or something like that. And a comment above the function as to what it does, something along the lines of "export the enviromnent variables for pipeline info"

Reply:
As a matter of fact, this function doesn't export any thing to env, it just generate some shell code which can be sourced later. I will add some comment here rather than changing the name, this will also make it clearer